packages/hono/README.md

```diff
@@ -0,0 +1,28 @@
+[![npm version](https://badge.fury.io/js/orval.svg)](https://badge.fury.io/js/orval)
```

Comment:
We should also add a link to the `hono` example in the `README` of other packages. but, It's not a direct effect, so you can deal with it later.

Reply:
Just updated the readme too